app/v1/connectors/RetrieveSelfEmploymentAnnualSummaryConnector.scala

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v1.connectors
+
+import config.AppConfig
+import javax.inject.{Inject, Singleton}
+import uk.gov.hmrc.http.HeaderCarrier
+import uk.gov.hmrc.http.HttpClient
+import v1.connectors.httpparsers.StandardDesHttpParser._
+import v1.models.request.retrieveSEAnnual.RetrieveSelfEmploymentAnnualSummaryRequest
+import v1.models.response.retrieveSEAnnual.RetrieveSelfEmploymentAnnualSummaryResponseBody
+
+import scala.concurrent.{ExecutionContext, Future}
+
+@Singleton
+class RetrieveSelfEmploymentAnnualSummaryConnector @Inject()(val http: HttpClient,
+                                                             val appConfig: AppConfig) extends BaseDesConnector {
+  def retrieveSEAnnual(request: RetrieveSelfEmploymentAnnualSummaryRequest)(
+                    implicit hc: HeaderCarrier,
+                    ec: ExecutionContext): Future[DesOutcome[RetrieveSelfEmploymentAnnualSummaryResponseBody]] = {
+
+    get(
+      uri = DesUri[RetrieveSelfEmploymentAnnualSummaryResponseBody](s"business/self-employment/${request.nino}/${request.businessId}/annual/${request.taxYear}")
```

Comment:
Two things

One is that the des Uri is wrong, needs to be 
`/income-source/nino/{nino}/self-employments/{incomeSourceId}/annual-summaries/{taxYear}`

Two, the parameter for taxYear needs to be in the correct des format.

In the DES Spec you'll see the format for taxYear is just the year (e.g. 2018) so you'll need to convert it from something like 2017-18 -> 2018. There should be examples within the other connectors